plugins/frontity-headtags/includes/class-frontity-headtags.php

```diff
@@ -107,8 +107,11 @@ public function clear_cache() {
 	 * @return array|mixed
 	 */
 	public function get_headtags( $key, $query ) {
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$ignore_cache = isset( $_GET['ignore_cache'] ) && 'true' === $_GET['ignore_cache'];
```

Comment:
I would go with `skip_cache` because it is used by the [WP REST Cache plugin](https://wordpress.org/plugins/wp-rest-cache/). That way, if someone is using that plugin, they can use `skip_cache=1` to skip both caches. They only support `skip_cache=1` but in our case I would support `skip_cache=true` too, just in case.

Reply:
Agree, I'll change it.

Reply:
@luisherranz, I've been checking how that plugin works, and it not only supports  `skip_cache=1` but `skip_cache=ANYTHING` (see [this line](https://plugins.trac.wordpress.org/browser/wp-rest-cache/trunk/includes/api/class-endpoint-api.php#L238)). Even `skip_cache=false` would be `true`. 😅

One thing we can do is to add the `skip_cache` param and make it work the same as the plugin you mention but also adding another param, like `head_tags_skip_cache`, that will overwrite the `skip_cache` param (and if it's set to `false` it will be `false`). That way we keep control of our plugin.

What do you think?

Reply:
Ok, total control 👍